packages/product-json-to-csv/test/main.spec.js

```diff
@@ -0,0 +1,625 @@
+import StreamTest from 'streamtest'
+import { oneLineTrim } from 'common-tags'
+import ProductJsonToCsv from '../src/main'
+import * as writer from '../src/writer'
+
+const streamTest = StreamTest.v2
+
+describe('ProductJsonToCsv', () => {
+  let productJsonToCsv
+  beforeEach(() => {
+    const logger = {
+      error: () => {},
```

Comment:
Have you tried out Jests auto mocking? You import what you want to mock. 

import logger from utils/logger

to then mock

jest.mock(utils/logger)

All functions on the logger are now jest.fn().

Reply:
Haven't tried it, but I'd love to